services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None) -> QuerySet:
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]